app/main/presenters/search_summary.py

```diff
@@ -266,4 +266,8 @@ def _mark_up_filter(filter):
                     filter_string = self.rules.add_filter_preposition(
                         filter_id=filter, filter_string=filter_string)
                     processed_filters.append(filter_string)
+                else:
+                    raise Exception(
+                        "Filter with id '{}' not found in filterRules.".format(filter)
```

Comment:
Rather than throwing could we patch up with `_mark_up_filter(filter)`, as with the case when `filterRules` is None?  Or is that mad?